definitions/private-law/json/CaseEventToComplexTypes/UploadAdditionalApplications/C2DocumentBundle/CaseEventToComplexTypes.json

```diff
@@ -28,20 +28,33 @@
     "CaseEventID": "uploadAdditionalApplications",
     "CaseFieldID": "temporaryC2Document",
     "ListElementCode": "documentAcknowledge",
-    "FieldDisplayOrder": 3,
+    "FieldDisplayOrder": "3",
     "DisplayContext": "MANDATORY",
-    "EventElementLabel": "Tick to confirm this document is related to Familyman Case number/Names/CCD"
+    "EventElementLabel": "Tick to confirm this document is related to this case"
   },
   {
     "LiveFrom": "20/11/2022",
     "CaseTypeID": "PRLAPPS",
     "ID": "C2DocumentBundleComplexType",
     "CaseEventID": "uploadAdditionalApplications",
     "CaseFieldID": "temporaryC2Document",
-    "ListElementCode": "c2AdditionalOrdersRequested",
+    "ListElementCode": "caReasonsForC2Application",
```

Comment:
this will have issue if any case already present with old element code?

Reply:
agreed, but this event was never enabled in prod, so should not be any issue.